packages/contracts-bedrock/src/dispute/AnchorStateRegistry.sol

```diff
@@ -30,8 +30,8 @@ contract AnchorStateRegistry is Initializable, ISemver {
     }
 
     /// @notice Semantic version.
-    /// @custom:semver 2.0.1-beta.6
-    string public constant version = "2.0.1-beta.6";
+    /// @custom:semver 2.0.1-beta.7
+    string public constant version = "2.0.1-beta.7";
```

Comment:
Why is this version bumped, what change was made here?

Reply:
I originally modified the ASR to add some additional functionality, but later reverted it. This escaped the revert. I've put the semver back to where it was before.